hugo/layouts/_default/index_flex.html

```diff
@@ -13,7 +13,7 @@ <h2 id="title" class="ml-2">Welcome to the ACL Anthology</h2>
           The ACL Anthology currently hosts {{ $.Scratch.Get "paper_count" }} papers on the study of computational linguistics and natural language processing.
         </div>
         <div class="card-body">
-          <a href="http://groups.google.com/group/acl-anthology?hl=en">Subscribe to the mailing list</a> to receive announcements and updates to the Anthology.
+          <a href="http://groups.google.com/group/acl-anthology?hl=en">Subscribe to the mailing list</a> or the <a href="{{ "/news/index.xml" | relURL }}">RSS news feed</a> to receive announcements and updates to the Anthology.
```

Comment:
"news/index.xml" should not begin with a slash, I believe (then the link would also work on the preview)

Reply:
Good catch. I only tested this locally and didn't think about the fact that the page might not be at the root of the domain. :see_no_evil: